Makefile

```diff
@@ -4,37 +4,37 @@ ifneq ("$(wildcard .env)","")
     export $(shell sed 's/=.*//' .env)
 endif
 
-.PHONY: build test coverage
+.PHONY: build test coverage clean
 
 # 154615699 corresponds to release v0.1.7 of Kakarot SSJ.
 KKRT_SSJ_RELEASE_ID = 154615699
 # Kakarot SSJ artifacts for precompiles.
 KKRT_SSJ_BUILD_ARTIFACT_URL = $(shell curl -L https://api.github.com/repos/kkrt-labs/kakarot-ssj/releases/${KKRT_SSJ_RELEASE_ID} | jq -r '.assets[0].browser_download_url')
-KATANA_VERSION = v1.0.0-alpha.11
+KATANA_VERSION = v1.0.0-alpha.0
```

Comment:
```suggestion
KATANA_VERSION = v1.0.0-alpha.11
```

revert

Reply:
I think it's on purpose to improve madara relative benchmark

Reply:
either way, we're going to bump this on Monday with the next katana nightly, we need to test the ec precompiles